### PR TITLE
(SIMP-8958) Standardize assets in pupmod-simp-simplib

### PR DESCRIPTION
**Pull request by op-ct**
This patch baselines the latest standardized assets for Puppet modules.

SIMP-9078 #close
[SIMP-8958] #comment Standardized assets in pupmod-simp-simplib
[SIMP-8839] #comment Removed EL6 from supported OSes in pupmod-simp-simplib
[SIMP-8489] #comment Updated pupmod-simp-simplib GLCI pipeline to Puppet 6.18
[SIMP-8923] #comment Renamed 'sanity' to 'releng' in pupmod-simp-simplib
[SIMP-8984] #comment Updated pupmod-simp-simplib to new GLCI conventions

[SIMP-8958]: https://simp-project.atlassian.net/browse/SIMP-8958
[SIMP-8839]: https://simp-project.atlassian.net/browse/SIMP-8839
[SIMP-8489]: https://simp-project.atlassian.net/browse/SIMP-8489
[SIMP-8923]: https://simp-project.atlassian.net/browse/SIMP-8923
[SIMP-8984]: https://simp-project.atlassian.net/browse/SIMP-8984

### CODE REVIEW

**Comment by lnemsick-simp** [comment]
Unit tests fail on Puppet 7.  Entered ticket: https://simp-project.atlassian.net/browse/SIMP-9115 .  Will configure GitLab job to allow failures.

File: Gemfile [hunk 1]
```diff
@@ -26,15 +32,11 @@ group :development do
...
   gem 'pry-doc'
-  gem 'rubocop'
-  gem 'rubocop-rspec'
-  gem 'rubocop-i18n'
 end
 
 group :system_tests do
   gem 'beaker'
   gem 'beaker-rspec'
-  gem 'beaker-windows'
```
Comment (lnemsick-simp) [inline Gemfile:37]: acceptance tests fail without this.  I will fix

**Comment by lnemsick-simp** [comment]
https://gitlab.com/simp/pupmod-simp-simplib/-/pipelines/241844768

**Comment by trevor-vaughan** [comment]
[This](https://gitlab.com/simp/pupmod-simp-simplib/-/jobs/964238220) is a valid failure due to the latest puppet 6+ switching from `master` to `server` and needs to have an adjustment made in the fact to allow either to be called.

Working on the fix now.

**Review by trevor-vaughan — APPROVED** [review]

**Pull request merged**